prompttools/benchmarks/load_data.py

```diff
@@ -0,0 +1,44 @@
+from datasets import load_dataset_builder,load_dataset,get_dataset_config_names, Dataset
```

Comment:
We'll want to guard `prompttools` from requiring `datasets` to be installed as part of the requirements.

Suggested change:

```
try:
    from datasets import load_dataset_builder, load_dataset, get_dataset_config_names, Dataset
    from datasets.dataset_dict import DatasetDict
except ImportError:
    load_dataset = None
```